README.md

```diff
@@ -35,8 +35,18 @@ In addition, you may need to set the following environment variables in your env
 |  LAUNCHABLE_TOKEN  |  (Required) A token to access Launchable API  |
 
 ## Development
-Pull requests are always appreciated. If you want to see whether your changes work as expected,  run the following command to install the plugin locally.
+Pull requests are always appreciated. Below are some tips on developing nose-launchable plugin. 
+
+### Install nose-launchable locally 
+If you want to see whether your changes work as expected,  run the following command to install the plugin locally.
 
 ```bash
-$ python setup.py develop
+$ python setup.py install
 ``` 
+
+### Release nose-launchable
```

Comment:
We should run these commands at the main branch after merged a feature branch to the main branch, right?

Reply:
Yes, that workflow was in my head when I wrote it down but you can actually include those changes into a feature branch if you want. It works in either way but `./scripts/bump_up.sh` creates a new branch so it does not work well with a feature branch...

Reply:
Thank you, I'll try it